cms/admin/forms.py

```diff
@@ -179,6 +179,8 @@ def __init__(self, *args, **kwargs):
             self.fields['navigation_extenders'].widget = forms.Select({}, [('', "---------")] + menu_pool.get_menus_by_attribute("cms_enabled", True))
         if 'application_urls' in self.fields:
             self.fields['application_urls'].choices = [('', "---------")] + apphook_pool.get_apphooks()
+        self.fields['redirect'].widget.attrs['style'] = "width:221px"
+        self.fields['overwrite_url'].widget.attrs['style'] = "width:221px"
```

Comment:
This feels like it's putting presentation logic into the validation layer. Can we instead change the width with a .css file for the page? Django's admin widget can be a little hard to target, but you should be able to pick out an ID for the widget and target it in a CSS file that you include in the admin's media.